### PR TITLE
Prepare handover gsi 112

### PR DESCRIPTION
**Pull request by MoritzHahn1337**
Prepare handover documentation

### CODE REVIEW

File: .description.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+All file-related metadata is stored in an internal mongodb database, owned and controlled
+by this service.
+It exposes no API enpoints and communicates with other services via events.
+
+### Events consumed:
+
+#### files_to_register
+This event signals that there is a file to register in the database.
+The file-related metadata from this event gets saved in the database and the file is
+pulled from the staging bucket dto the permanent storage.
```
Comment (mephenor) [inline .description.md]:
```suggestion
moved from the incoming staging bucket to the permanent storage.
```

File: .description.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+#### files_to_stage
+This event signals that there is a file that needs to be staged for download.
+The file is then copied over from the permanent storage to the outbox.
```
Comment (mephenor) [inline .description.md]:
```suggestion
The file is then copied from the permanent storage to the outbox for the actual download.
```

File: .description.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+<!-- Please provide a short overview of the features of this service.-->
+
+This service provides an infrastructure to administer files stored in an S3-compatible
```
Comment (mephenor) [inline .description.md]:
```suggestion
This service provides functionality to administer files stored in an S3-compatible
```

**Review by mephenor — CHANGES_REQUESTED** [review]
Smaller description changes

File: .description.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+<!-- Please provide a short overview of the features of this service.-->
+
+This service provides functionality to administer files stored in an S3-compatible
+object storage.
+All file-related metadata is stored in an internal mongodb database, owned and controlled
+by this service.
+It exposes no API enpoints and communicates with other services via events.
```
Comment (KerstenBreuer) [inline .description.md]:
```suggestion
It exposes no REST API enpoints and communicates with other services via events.
```

**Review by mephenor — DISMISSED** [review]

**Review by KerstenBreuer — DISMISSED** [review]

**Review by mephenor — APPROVED** [review]

**Pull request merged**